### PR TITLE
Code: Explore > Reconciliation

### PR DESCRIPTION
**Pull request by gemfarmer**
For #1193 

Preview: https://federalist.18f.gov/preview/18F/doi-extractives-data/reconciliation/explore/reconciliation/

What i have done:
- [x] added file to parse data from [here](https://docs.google.com/spreadsheets/d/1gacj2fFQF-5cdxVoxbdOpytBRfdY-Ujfq-ZGPblNc2Y/edit#gid=165381653) to workable format 
- [x] parsed data. Can be viewed [here](https://github.com/18F/doi-extractives-data/blob/reconciliation/data/reconciliation/revenue.tsv)
- [x] update html / client-side js to work with that data, using revenue-by-company as a guide. 

As of https://github.com/18F/doi-extractives-data/commit/bed11279ac9392ff47f4c37b6dca6a3c46aff8a8 TODO:

Clean up:
* [x] address new hound errors
* [x] remove +/- variance in summary 
* [x] address line notes, namely removing unused files

Totals:
* [x] sort values (highest to lowest variance)
* [x] make sure extent values are accurate
* [x] `NaN` bug
* [x] display `DNR`,`DNP`, `N/A`
* [x] %s should be absolute values

Company List area:
* [x] remove extra heading
* [x] make table look half decent
* [x] `NaN` bug
* [x] display `DNR`,`DNP`, `N/A`
* [x] %s should be absolute values
* [x] "material" variance

General:
* [x] Clean up code / address hound
* [x] check accuracy throughout process
* [x] remove negative % values
* [x] revisit Corporate Income Tax data
* [x] glossary link is clickable

### CODE REVIEW

**Comment by gemfarmer** [comment]
@mhz @ericronne Hey I think reconciliation is basically there. Could you two take a look and tell me what you think?

https://federalist.18f.gov/preview/18F/doi-extractives-data/reconciliation/explore/reconciliation/

Also, pretty much all of the hound errors are asking me to add semicolons where I shouldn't (there are chained functions which __can't__ have semicolons, so you can ignore them.

**Comment by ericronne** [comment]
Whoa that's a lotta content. Let's chat @gemfarmer ! :)

**Comment by gemfarmer** [comment]
@ericronne A lot of content on the left side? Agreed. I think a lot of it should probably go. Waiting for feedback from @mhz and @coreycaitlin. Can talk any time this afternoon.

File: _explore/revenue/federal-revenue-by-company.html [hunk 1]
```diff
@@ -43,14 +43,14 @@ <h1 data-toggler="filters" data-filter-description="" class="filter-description
         </div>
 
 
-        <div class="container-left-6">
+        <!-- <div class="container-left-6">
```
Comment (gemfarmer) [inline _explore/revenue/federal-revenue-by-company.html]: making note to uncomment this

File: _explore/revenue/reconciliation.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Reconciliation | Explore Data
```
Comment (gemfarmer) [inline _explore/revenue/reconciliation.md]: this file is no longer necessary. remove it.

File: js/pages/company-revenue.js [hunk 3]
```diff
@@ -56,7 +56,8 @@
   var filters = root.selectAll('.filters [name]')
     .on('change', filterChange);
 
-  var search = root.select('#company-name-filter')
+  var search = root.select('#company-name-filter');
```
Comment (gemfarmer) [inline js/pages/company-revenue.js]: revert back to previous syntax

File: js/pages/recon.js [hunk 4]
```diff
@@ -0,0 +1,311 @@
+// globals d3, eiti, EITIBar
+(function() {
+  // 'use strict';
```
Comment (gemfarmer) [inline js/pages/recon.js]: remove this page. it is no longer necessary

**Comment by coreycaitlin** [comment]
I revised the content, and got it down to a few very brief paragraphs on the left side. Happy to walk through the details/rationale more thoroughly if there are questions.

**Comment by coreycaitlin** [comment]
The one recommendation I didn't implement here is that I think we should spell out "DNR/DNP" in the tables as "Did not report." If there's a significant difference, we should tease that out and clarify. 

@gemfarmer could you either implement that or show me how? (Maybe after we double-check the DNR/DNP difference?)

**Comment by meiqimichelle** [comment]
Whew! Now that I was able to FINALLY make a push to staging, I think we should merge this branch to `dev` and continue working in smaller branches to get the remaining issues. What say you, @gemfarmer @coreycaitlin @ericronne ? What I mean is -- I think this is a good enough first version to iterate from in development.

**Comment by ericronne** [comment]
Tons better, yay @coreycaitlin & @gemfarmer!

Could we...
- Add space above "Median payment…" header, to match similar pp, eg [this one] (https://useiti.doi.gov/explore/federal-revenue-by-company/)
- Center "payments reported by…" header

**Comment by meiqimichelle** [comment]
I moved @ericronne 's note to a separate issue #1234. Let's resolve these merge conflicts and get this work merged to `dev` @gemfarmer

**Comment by gemfarmer** [comment]
Rebased and merging!

**Pull request merged**